pennylane/transforms/decompositions/clifford_t_transform.py

```diff
@@ -411,44 +411,11 @@ def circuit(x, y):
                     d_ops = _two_qubit_decompose(op)
                     decomp_ops.extend(d_ops)
 
-                # For special multi-qubit gates and ones constructed from matrix
+                # If we don't know how to decompose the operation
                 else:
-                    try:
-                        # Attempt decomposing the operation
-                        md_ops = op.decomposition()
```

Comment:
Specifically this line, if this exists then it would have already been handled in `compile`.